app/helpers/answers_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AnswersHelper
+end
```

Comment:
![screen shot 2015-04-09 at 4 24 52 pm](https://cloud.githubusercontent.com/assets/3419757/7076103/12138f6a-ded5-11e4-84ac-b0a6eb9be99e.jpg)
This is totally a card in our Trello board